scene/gui/base_button.h

```diff
@@ -54,6 +54,10 @@ class BaseButton : public Control {
 	bool shortcut_feedback = true;
 	Ref<Shortcut> shortcut;
 	ObjectID shortcut_context;
+	Vector2 last_drag_position;
+	Vector2 drag_from;
+	float pressed_tolerance;
```

Comment:
```suggestion
	float pressed_tolerance; // Calculated in constructor.
```

Reply:
`// Calculated in constructor and NOTIFICATION_WM_DPI_CHANGE.`